cmd/titus/github.go

```
@@ -217,6 +229,25 @@ func runGitHubScan(cmd *cobra.Command, args []string) error {
+		if githubJitter > 0 {
+			cloneEnum.Jitter = time.Duration(githubJitter * float64(time.Second))
+		}
+
+		// Estimate scan time and prompt for confirmation if delay/jitter is enabled
+		if cloneEnum.Jitter > 0 || cloneEnum.Delay > 0 {
+			estimate := cloneEnum.EstimateScanTime()
+			if estimate > 0 {
+				_, _ = fmt.Fprintf(cmd.ErrOrStderr(), "Estimated scan time: %s (with %d repos)\n", formatDuration(estimate), len(repos))
+				_, _ = fmt.Fprintf(cmd.ErrOrStderr(), "Continue? [Y/n] ")
+				var response string
+				fmt.Scanln(&response)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Scanln` is not checked

(errcheck)